pkgs/applications/science/astronomy/galario/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv
+, fetchurl
+, cmake
+, fftw
+, fftwFloat
+, enablePython ? false
+, pythonPackages
+, llvmPackages
+}:
+
+stdenv.mkDerivation rec {
+  name = "galario-${version}";
+  version = "1.2.1";
+
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/mtazzari/galario/archive/v${version}.tar.gz";
+      sha256 = "1m1lc6z5db6h2l719080jlh2aaaa82p1ddfzsv0kqlh6biyx6psq";
+    })
+    (fetchurl {
+      url = "https://github.com/UCL/GreatCMakeCookOff/archive/v2.1.9.tar.gz";
```

Comment:
Can you please explain what you add this for?

Reply:
These are cmake recipes that are needed to build galario. I don't that other packages use them, so I didn't create a separate package for it. Instead I merely copy them in the source directory. 

I've added a short comment in the package.

Reply:
But [they](https://mtazzari.github.io/galario/install.html) don't mention this.
Does it not build without these recipes?

So what is the point of this? Do you think they're useful for other packages as well?

Reply:
They don't mention this in the documentation because the recipes are downloaded if `cmake` does not find them (see [CMakeLists.txt](https://github.com/mtazzari/galario/blob/89b9bef3ebdf1c5c4af999266ab45870fd478fa7/CMakeLists.txt#L43) and [cmake/LookUp-GreatCMakeCookOff.cmake](https://github.com/mtazzari/galario/blob/master/cmake/LookUp-GreatCMakeCookOff.cmake)). I could not find any other package in `nixpkgs` that use this.